pkgs/applications/misc/pydf/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "pydf";
+  version = "12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "7f47a7c3abfceb1ac04fc009ded538df1ae449c31203962a1471a4eb3bf21439";
+  };
+
+  meta = with stdenv.lib; {
+    description = "colourised df(1)-clone";
+    homepage = http://kassiopeia.juls.savba.sk/~garabik/software/pydf/;
+    license = licenses.publicDomain;
+    maintainers = with maintainers; [ monsieurp ];
```

Comment:
Looks like we'll need you to add yourself to the maintainer list, right about here: https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L3220 check out the top of the file for instructions and more info :)

Reply:
should I add myself right now?

Reply:
Yep! If you're good with git, it would be best to put it _before_ this commit, but if you don't know how, it isn't a big deal -- just add yourself in a second commit.

Reply:
come on ;) I know how to perform a rebase :) what should be the commit message?

Reply:
Something like `maintainers: add monsieurp`. Your entry should look something like:

```
monsieurp = {
     ...
}
```

_(not everybody knows how to rebase, is why I said that :))_

Reply:
done :)